alphadia/raw_data/alpharaw_wrapper.py

```diff
@@ -176,30 +176,16 @@ def __init__(self, raw_file_path: str):
 class Thermo(AlphaRaw, ThermoRawData):
     """Class holding pre-processed Thermo raw data."""
 
-    def __init__(
-        self, raw_file_path: str, process_count: int = 10, astral_ms1: bool = False
-    ):
+    def __init__(self, raw_file_path: str, process_count: int = 10):
         AlphaRaw.__init__(self)
         ThermoRawData.__init__(self, process_count=process_count)
         self.load_raw(raw_file_path)
-        self._preprocess_raw_data(astral_ms1)
-
-    def _filter_spectra(self, astral_ms1: bool = False):
-        """Filter the spectra for MS1 or MS2 spectra."""
-        # filter for Astral or Orbitrap MS1 spectra
-        if astral_ms1:
-            self.spectrum_df = self.spectrum_df[self.spectrum_df["nce"] > 0.1]
-            self.spectrum_df.loc[self.spectrum_df["nce"] < 1.1, "ms_level"] = 1
-            self.spectrum_df.loc[self.spectrum_df["nce"] < 1.1, "precursor_mz"] = -1.0
-            self.spectrum_df.loc[
-                self.spectrum_df["nce"] < 1.1, "isolation_lower_mz"
-            ] = -1.0
-            self.spectrum_df.loc[
-                self.spectrum_df["nce"] < 1.1, "isolation_upper_mz"
-            ] = -1.0
-        else:
-            self.spectrum_df = self.spectrum_df[
-                (self.spectrum_df["nce"] < 0.1) | (self.spectrum_df["nce"] > 1.1)
-            ]
+        self._preprocess_raw_data()
+
+    def _filter_spectra(self):
+        """Filter the spectra for collision energies < 0.1 and > 1.1 and create `spec_idx`."""
```

Comment:
I think we can drop this function entirely as it should always be a no-op. This else was only needed for the special data where we ecnoded ms1 by passing collision energy as 1.0